contents/node_operator.rst

```diff
@@ -632,8 +632,10 @@ General configuration parameters
    "SwapEnabled","--swap","SWARM_SWAP_ENABLE","false","Enable SWAP"
    "SwapPaymentThreshold","--swap-payment-threshold","SWARM_SWAP_PAYMENT_THRESHOLD","1000000","honey amount at which payment is triggered"
    "SwapDisconnectThreshold","--swap-disconnect-threshold","SWARM_SWAP_DISCONNECT_THRESHOLD","1500000","honey amount at which a peer disconnects"
-   "SwapInitialDeposit","--swap-initial-deposit","SWARM_INITIAL_DEPOSIT","0","Initial deposit amount for swap chequebook"
+   "SwapDepositAmount","--swap-deposit-amount","SWARM_SWAP_DEPOSIT_AMOUNT","0","Deposit amount for swap chequebook"
+   "SwapSkipDeposit", "swap-skip-deposit", "SWARM_SWAP_SKIP_DEPOSIT", "false", "on't deposit during boot sequence"
    "SwapLogPath","--swap-audit-logpath","SWARM_SWAP_LOG_PATH","n/a","Write execution logs of swap audit to the given directory"
+   "SwapChequebookFactory", "swap-cheqeubook-factory", "SWARM_SWAP_CHEQUEBOOK_FACTORY_ADDR", "defined per blockchain network", "SWAP chequebook factory contract address"
```

Comment:
"defined per blockchain network" seems to be the default here, which doesn't make much sense?

Reply:
There is a default variable on Ropsten, and we can define different default variables on different networks. 
Do you have a suggestion on how to correctly describe this here?

Reply:
"Defined per blockchain network" should go into the last bit, the actual description of the variable.

The fourth entry is the **Default value**, so it should contain at least a valid entry, not a description. Some variables already have `n/a` in that column (not applicable), in this case I would put the default value for ropsten

But the real point here is that I think there is a lack of documentation explaining *why* we need these variables and *when* to set them. Someone without context may well have difficulties understanding what this all is about.

I think this means we need to start writing a documentation about **swap** in these docs as well. A separate chapter describing how it works, prerequisites, how to set it up etc.

For this PR I suggest:
* Add the network ID 5 factory as default
* In the last column, explain that the value is different for each blockchain.

Reply:
I agree with your suggestion about creating a SWAP documentation. Where do we create an issue for this? 

> Add the network ID 5 factory as default

Network ID 5 is about the *Swarm* network ID, the factory default is different per *Blockchain* network ID (I understand the confusion).